src/direct-auth-idx/direct-auth-idx/AuthenticationHelper.cs

```diff
@@ -27,7 +27,17 @@ public static async Task<ClaimsIdentity> GetIdentityFromAuthResponseAsync(IdxCon
             return identity;
         }
 
-        private static async Task<IEnumerable<Claim>> GetClaimsFromUserInfoAsync(IdxConfiguration configuration, string accessToken)
+        public static Claim[] GetClaimsFromTokenResponse(ITokenResponse tokenResponse)
```

Comment:
This code is duplicated. We already have a method that get claims from userinfo + attach tokens. Please, let's reuse that: 
*https://github.com/okta/okta-idx-dotnet/blob/idx-auth-layer/src/direct-auth-idx/direct-auth-idx/AuthenticationHelper.cs#L20
* https://github.com/okta/okta-idx-dotnet/blob/idx-auth-layer/src/direct-auth-idx/direct-auth-idx/Controllers/AccountController.cs#L60

Reply:
I considered using that method but I can't use because at the point that I need to make the call I don't have an AuthenticationResponse only a TokenResponse.